metadata.yaml

```diff
@@ -20,7 +20,8 @@ provides:
 storage:
   mongodb:
     type: filesystem
-    location: /data/db
+    # mongod is strictly confined in the charmed-mongodb snap
+    location: /var/snap/charmed-mongodb/common/db
```

Comment:
Location in snap should match the spec - https://docs.google.com/document/d/1MyT4Wt_-vgPwhwEjC-oeHRdLYrZWaQzFGTwclT7hD1U/edit

Reply:
Also I think the config file should change as well